sdk/endpoints.go

```diff
@@ -312,11 +323,61 @@ func GetReceipt(delegateNode types.Node, hash string) (*types.Receipt, error) {
 	return &transaction.Receipt, nil
 }
 
+// GetTransactions - Get details about sent transactions for a node
+func GetTransactions(delegateNode types.Node, page, pageStart, pageSize string) ([]types.Transaction, error) {
```

Comment:
Shouldn't page and pageSize be int of some type?

Reply:
They are passed into the curl command so i left them as strings. The server that receives them should change them into ints if its proper. I could force check to make sure its int convertible tho.